### PR TITLE
Update 3113

### PR DESCRIPTION
**Pull request by jfbastien**

### CODE REVIEW

File: test/torture-s/ashrdi-1.c.s [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+	block
+	block
+	block
+	block
+	block
+	block
+	block
+	block
+	block
+	block
```
Comment (jfbastien) [inline test/torture-s/ashrdi-1.c.s]: This is funny.

**Comment by kripken** [comment]
Why are there new failures?

**Comment by jfbastien** [comment]
New tests which didn't make it to binaryen before the update. They're new in this PR.

**Comment by kripken** [comment]
Why are they new? I thought the torture test set was fixed? Or are they new to not crash the backend?

**Comment by jfbastien** [comment]
The set of torture sets is currently pinned by the waterfall: https://github.com/WebAssembly/waterfall/blob/f18e24da6a5bad33816a6c3dc15b01f4cb75977b/src/build.py#L135
I figured I'd only update that set from time to time (it doesn't change often anyways).

Some of the tests don't make it through the pipeline though.

In this case @sunfishcode fixed an LLVM bug here: http://llvm.org/viewvc/llvm-project?view=revision&revision=260737
and @dschuff updated the expected LLVM failures accordingly here: http://llvm.org/viewvc/llvm-project?view=revision&revision=260750

The tests could also fail s2wasm and not make it to binaryen-shell, or they could fail sexpr-wasm and not make it to d8 and spec.

**Pull request merged**